myriad-scheduler/src/main/java/com/ebay/myriad/state/SchedulerState.java

```diff
@@ -43,15 +49,18 @@
     private Set<Protos.TaskID> lostTasks;
     private Set<Protos.TaskID> killableTasks;
     private MyriadState myriadState;
+    private Protos.FrameworkID frameworkId;
+    private RMStateStore stateStore;
 
-    public SchedulerState(MyriadState myriadState) {
+    public SchedulerState(RMStateStore stateStore) {
```

Comment:
Plan to change the input param to MyriadStateStore to keep it agnostic of the implementation